.github/workflows/scripts/verify_tag.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+from os import environ
+from pathlib import Path
+
+# See https://docs.github.com/en/actions/reference/environment-variables
+ENV_VARIABLE = 'GITHUB_REF'
+
+
+def go():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('folder', help='Directory where to look for wheels and source dist')
```

Comment:
Why wouldn't call the argument `path` or `directory`? `folder` smells Windows :slightly_smiling_face:

Reply:
I started with `dir` but it shadows the built, and directory was too long.